Selector: Fix query being valid with incomplete matchers (#23)
grafana/lezer-logql: test/expression.txt

==>

LogQL(Expr(LogExpr(Selector(Matchers(Matcher(Identifier, Eq, String))), PipelineExpr(PipelineExpr(PipelineStage(LineFilters(LineFilter(Filter(PipeExact), String)))), PipelineStage(Pipe, JsonExpressionParser(Json, JsonExpressionList(JsonExpressionList(JsonExpressionList(JsonExpression(Identifier, Eq, String)), JsonExpression(Identifier, Eq, String)), JsonExpression(Identifier, Eq, String))))))))

> # Simple log query with back-ticks and comma to have error

Reviewer: ```suggestion
# Simple log query with trailing comma in stream selector to have error
```

Author: Thank you, that is a much better name! Fixed in [b0e30e2](https://github.com/grafana/lezer-logql/pull/23/commits/b0e30e24940b6058291cbccb01b9118bef70f913).
